libs/geo/src/test/java/org/elasticsearch/geometry/utils/SpatialEnvelopeVisitorTests.java

```diff
@@ -48,7 +49,7 @@ public void testVisitGeoShapeNoWrap() {
     public void testVisitGeoShapeWrap() {
         for (int i = 0; i < 1000; i++) {
             var geometry = GeometryTestUtils.randomGeometryWithoutCircle(0, true);
-            var bbox = SpatialEnvelopeVisitor.visitGeo(geometry, false);
+            var bbox = SpatialEnvelopeVisitor.visitGeo(geometry, WrapLongitude.NO_WRAP);
```

Comment:
I think this should be wrapping, according to the test method name.

Reply:
Then the test itself needs to change because it's failing when it's set to `WRAP`

Reply:
OK. Perhaps add a TODO comment.